### PR TITLE
Refactoring Foundation article and adding new content

### PR DESCRIPTION
**Pull request by seriouslysean**

### CODE REVIEW

**Comment by pawan-mehta** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  ossbackup2@gmail.com
    Domain oss-india.net has exceeded the max emails per hour (100/100 (100%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from mail-oi0-f54.google.com ([209.85.218.54]:50006)
	by server14.hostingraja.in with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.89)
	(envelope-from <pawan.ossindia+caf_=pawan=oss-india.net@gmail.com>)
	id 1e5Fz6-000TGf-Uj
	for pawan@oss-india.net; Fri, 20 Oct 2017 00:26:15 +0530
Received: by mail-oi0-f54.google.com with SMTP id w197so16520949oif.6
        for <pawan@oss-india.net>; Thu, 19 Oct 2017 11:55:39 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:delivered-to:dkim-signature:dkim-signature:date
         :from:reply-to:to:cc:message-id:subject:mime-version
         :content-transfer-encoding:precedence:list-id:list-archive:list-post
         :list-unsubscribe;
        bh=BETmG0B6pafoOOMEaZZzJIY4YINbtbAImGZ6ioPQ2s4=;
        b=GBKriBwjMzy192sY5E+tP6/nxJyOOOHmCV+IYrVGlkZD+OJMaTumMp+e2e8k9tM4vj
         PqOIUeoVGDUfQQTMNhWSPPc0c2AVuOWpOlaQZFpbOsdIdeTHOkFZcFndPCq+XNdCefXg
         gdZWxaaUzTPHxtM0AgAC8hk6D/zlIUp/eSFDS2a9AglV7xNGbligkZy4zIs7B4VPiUIA
         TZ/kAA3B5c3BQeDLGP77SLUZL3/iDwLLEy4LTk+TjLYEp3I8w6htWkILdYav8CG7i2uP
         9qVE4EIMA8S6TmpIUZbRs5wo+Xa1ER0r7wGRTkZGZU5kVso+gC3aoGwaQcCgVBBTzpiY
         FZZQ==
X-Gm-Message-State: AMCzsaXXVgVj8OVqphg6fj7d99pVuqUgTpWcqSSHPckPZtIziMYYOZXC
	ZGxslU74bxGf1TDlRCOIdhud3loYwRgg0/X6eQLJbkYH7CwE5BQ=
X-Received: by 10.157.38.238 with SMTP id i43mr1517090otd.94.1508439317210;
        Thu, 19 Oct 2017 11:55:17 -0700 (PDT)
X-Forwarded-To: pawan@oss-india.net
X-Forwarded-For: pawan.ossindia@gmail.com pawan@oss-india.net
Delivered-To: pawan.ossindia@gmail.com
Received: by 10.168.68.11 with SMTP id r11csp4211625ofa;
        Thu, 19 Oct 2017 11:55:16 -0700 (PDT)
X-Google-Smtp-Source: ABhQp+RPpk81AuuYC6lokkWXzpcuXNT2+Sw3JrAsPLfwxogCrI/OF9mlGJn7DtCa5FVvwzkFBsKV
X-Received: by 10.107.197.132 with SMTP id v126mr2986406iof.279.1508439316441;
        Thu, 19 Oct 2017 11:55:16 -0700 (PDT)
ARC-Seal: i=1; a=rsa-sha256; t=1508439316; cv=none;
        d=google.com; s=arc-20160816;
        b=wF+1KkF2plyM3YN6AYYQTCvhyI366lKyTGrcu+4hDfTHhIvrZJ+7IFq+mpsFszPZDN
         19cczUePSrJONP7svfy9gxespGkeNsSXHgNtc1fQpOzSOezkdkcTpqG+1Ii5b61wSsXz
         wUYsftlOtFYqYVr/B7vYQknnRzHZBjs2SfWJfrV3e01Rw0wDEErs7v59i9f/CMjMtS1E
         dSWRMFus7Q6M+eGwwwB7wsmNu1G9yWLfLuZIgmbWa4W8WO0Qf7Z7oE7iuF4q71hKycpC
         3vmIwmLJZODyqsWhCuPUScGb9WdCFCVhnXRDHqKYiZe/V1lvBWoAm6bTOq3JWEaxPYWl
         h3ZA==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=list-unsubscribe:list-post:list-archive:list-id:precedence
         :content-transfer-encoding:mime-version:subject:message-id:cc:to
         :reply-to:from:date:dkim-signature:dkim-signature
         :arc-authentication-results;
        bh=BETmG0B6pafoOOMEaZZzJIY4YINbtbAImGZ6ioPQ2s4=;
        b=X2RB3COiETXV4WGFG3VBJjLkyAr0jd+a+oih0/1XRAwLd9x7kmpA+96W+w80BM+57E
         uQQFH/SeZEAGKMnlOPQRWwqSEnpVT3fHKBKcYEqhKErZ6iStWhhdtO9q2fO85PA4WMlf
         8BhcRnNTK9VC2ujYzxib4FrNDgjGj9BNwvIdO2WMefabeXVFGutAgB1VANK+jOnmUMoD
         ON/S4pZXFFp8X03OuvI67Q6tOOJ1GtyVe0+9LzcHIqrbIP2/IBZ5e7XfJiucF78I/sj5
         Mq8Y+st7CrRIuZN5PUUrl0qsQGO5oU25gkAMAu12ysxBxuOQK6kv13BRRYpT0N7AJ9Gb
         CO8w==
ARC-Authentication-Results: i=1; mx.google.com;
       dkim=pass header.i=@github.com header.s=s20150108 header.b=SdnJIHZj;
       dkim=pass header.i=@sendgrid.info header.s=smtpapi header.b=xy9CfxWk;
       spf=pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 192.254.113.101 as permitted sender) smtp.mailfrom=bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com;
       dmarc=pass (p=NONE sp=NONE dis=NONE) header.from=github.com
Received: from o6.sgmail.github.com (o6.sgmail.github.com. [192.254.113.101])
        by mx.google.com with ESMTPS id w66si3543233ioi.261.2017.10.19.11.55.16
        for <pawan.ossindia@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Thu, 19 Oct 2017 11:55:16 -0700 (PDT)
Received-SPF: pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 192.254.113.101 as permitted sender) client-ip=192.254.113.101;
Authentication-Results: mx.google.com;
       dkim=pass header.i=@github.com header.s=s20150108 header.b=SdnJIHZj;
       dkim=pass header.i=@sendgrid.info header.s=smtpapi header.b=xy9CfxWk;
       spf=pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 192.254.113.101 as permitted sender) smtp.mailfrom=bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com;
       dmarc=pass (p=NONE sp=NONE dis=NONE) header.from=github.com
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com; 
	h=from:reply-to:to:cc:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=nBY3ybIkHQAIDOQ1hPZTB2c1YaU=; b=SdnJIHZj7UosoB2r
	Th/Rvg2+aL8dLkmyj/qs2HddMJGMVSOCpJoRjfqLtFM7eIKfb1BJHe2BIBjWpf2r
	lUtXOLbIOtN/EjCSah9jdY6qWEHuUtXmgY0AmNCN1VnJmdutisEHyr2pWXKlPvF3
	+X8rHC6OZefbqVHrbBnttmps0zA=
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=sendgrid.info; 
	h=from:reply-to:to:cc:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe:x-feedback-id; 
	s=smtpapi; bh=nBY3ybIkHQAIDOQ1hPZTB2c1YaU=; b=xy9CfxWkThYipdUCRS
	5TUm0FvYBLfxXA2NcZ38ntUyRa1mQxBuMG6puYpz9xEQGNYbiDgRgLijVGrFydLk
	96vSJ+0KiekROgkwvPCeXBjroiNpBpDwZuj5mgndlTitiJnaQzXah9qDqxX5Mxbg
	KzqxjqCfyprz3tQSzJrDxRUB4=
Received: by filter0560p1mdw1.sendgrid.net with SMTP id filter0560p1mdw1-18143-59E8F511-28
        2017-10-19 18:55:13.59806158 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd0002p1iad1.sendgrid.net (SG) with ESMTP id 9kBJ0YbLRKWk86hK1Yn5Dg
	for <pawan.ossindia@gmail.com>; Thu, 19 Oct 2017 18:55:13.380 +0000 (UTC)
Date: Thu, 19 Oct 2017 18:55:16 +0000 (UTC)

**Comment by seriouslysean** [comment]
@johnkennedy9147 all good 👍

File: src/pages/css/css-frameworks/css-framework-foundation/index.md [hunk 1]
```diff
@@ -25,13 +26,12 @@ Here is a simple HTML template which includes the latest compiled and minified C
       </body>
     </html>
 ```
- <kbd>[![](//discourse-user-assets.s3.amazonaws.com/original/2X/f/f9c81b35877c3e0551ca461c75b78aac80c6f48a.png) **JSBin Demo**](http://jsbin.com/gebolikiru/edit?html,output)</kbd>
 
-We have used a CDN in this example, but you can checkout other ways of installing Foundation <a href='http://foundation.zurb.com/sites/download/' target='_blank' rel='nofollow'>here</a>.
+This example makes use of a CDN that loads the default settings. If you'd like to customize the grid layout, change the colors or add and remove components you can do so on the Foundation <a href='http://foundation.zurb.com/sites/download/' target='_blank' rel='nofollow'>download</a> page.
```
Comment (johnkennedy9147) [inline src/pages/css/css-frameworks/css-framework-foundation/index.md]: @seriouslysean missed one :-)

**Comment by seriouslysean** [comment]
@johnkennedy9147 I can't even claim that it's Monday morning, either, haha. All fixed!

**Pull request merged**